R/template_data_frame.R

```diff
@@ -25,7 +28,7 @@ template_data_frame_categories <- function(dat_dict, numcases, varnames, survey
     dis_output[[i]] <- sample(vals, numcases, replace = TRUE)
   }
 
-  multivars <- dat_dict[[varnames]][dat_dict$data_element_valuetype == "MULTI"]
```

Comment:
This throws a warning with the outbreak dictionaries as they have not been moved to the Kobo system AFAICT